server/game/gamesteps/challenge/challengeflow.js

```diff
@@ -17,6 +17,7 @@ class ChallengeFlow extends BaseStep {
             new SimpleStep(this.game, () => this.announceChallenge()),
             new SimpleStep(this.game, () => this.promptForAttackers()),
             new SimpleStep(this.game, () => this.announceAttackerStrength()),
+            new SimpleStep(this.game, () => this.chooseStealthTargets()),
```

Comment:
Per discussion earlier in Discord, misread this and everything is still in the right order. Sorry.